### DISCUSSION TITLE
To get someone's presence (if they're typing or online)

### DISCUSSION THREAD

**Discussion by MsccoderCalix** [WhiskeySockets/Baileys#269] [General]
how can i get the presence of someone? i use  await sock.presenceSubscribe("xyz@s.whatsapp.net")  but it return undefined.
some of you find other alternative

**Comment by shabika0A** [comment]
I guess if the user abandons last seen for you , baileys will not send you online event.

**Reply by MsccoderCalix** [reply]
Follow me please shabika
